manifests/init.pp

```diff
@@ -126,5 +132,33 @@
       group  => $puppet_group,
       mode   => '0750',
     }
+    if $manage_fact_watcher_dependencies {
+      $ensure = if $fact_watcher_service {
+        'installed'
+      } else {
+        'absent'
+      }
+      package { 'ruby-inotify':
+        ensure   => 'installed',
+        provider => 'puppet_gem',
```

Comment:
I dislike using the `puppet_gem` provider (or really, `gem` on production servers). If you upgrade your Puppet AIO it may break the native extensions and you don't know what you're pulling in.